MagnetScannerIOS/NetworkResolver.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  NetworkResolver.swift
+//  Magnet
+//
+//  Created by Francisco Jordano on 20/10/2016.
+//
+
+import Foundation
+import Alamofire
+import SwiftyJSON
+
+//
+// In charge of fetching the urls associated to the latitude and longitude
+// passed.
+//
+public class NetworkResolver {
+  static let API_END_POINT: String = "https://tengam.org/content/v1/search/beacons/"
+  static let SLUG_BASE_URL: String = "https://tengam.org/"
+  static let RADIUS: Int = 1500
```

Comment:
Cause I needed that to see something from my place.

Will reduce it to 50 as the android one.